### DISCUSSION TITLE
Feature : Screens for generated INSERT, UPDATE, SELECT Statements

### DISCUSSION THREAD

**Discussion by nklamann** [pawelsalawa/sqlitestudio#3916] [Q&A]
### Details
I would like to have screens with generated SQL Statements (INSERT/UPDATE/SELECT  similar to other tools.

DBeaver for example (which is in all other aspects less comfortable than SQLiteStudio for sqlite) generates these statements.

See attached screenshots
![generated_insert](https://user-images.githubusercontent.com/1001597/104715122-646a6480-5726-11eb-9676-73e8b2bcd234.png)
![Dbeaver Menu](https://user-images.githubusercontent.com/1001597/104715124-6502fb00-5726-11eb-8c59-ba91d3e26c0e.png)

If a similar feature already exists : Please accept my apology !

### Operating system
Windows 10

**Comment by pawelsalawa** [comment]
How about this?

![image](https://user-images.githubusercontent.com/481846/104783301-02d9e280-5786-11eb-8bd9-9adc02c3e070.png)

![image](https://user-images.githubusercontent.com/481846/104783318-0cfbe100-5786-11eb-8234-92ce694393ba.png)
